src/test/java/io/confluent/examples/streams/CustomStreamTableJoinIntegrationTest.java

```diff
@@ -191,24 +191,24 @@ public void shouldTriggerStreamTableJoinFromTable() throws Exception {
     final Duration frequencyToCheckForExpiredWaitTimes = Duration.ofSeconds(2);
 
     final List<KeyValueWithTimestamp<String, Double>> inputStreamRecords = Arrays.asList(
-      new KeyValueWithTimestamp<>("alice", 999.99, TimeUnit.MILLISECONDS.toMillis(10)),
-      new KeyValueWithTimestamp<>("bobby", 222.22, TimeUnit.MILLISECONDS.toMillis(15)),
-      new KeyValueWithTimestamp<>("alice", 555.55, TimeUnit.MILLISECONDS.toMillis(30)),
-      new KeyValueWithTimestamp<>("alice", 666.66, TimeUnit.MILLISECONDS.toMillis(40))
+        new KeyValueWithTimestamp<>("alice", 999.99, TimeUnit.MILLISECONDS.toMillis(10)),
```

Comment:
Seem the new auto-formatting setting is messing with us :)

Why does it apply 4 space here?

Reply:
That's because it's a continuation of the previous line, and in this case the new auto-formatting says "Use 4 spaces (before: 2 spaces) in this situation".